### PR TITLE
Removes alpha validation from last name

### PR DESCRIPTION
**Pull request by chloealee**
#### What's this PR do?
- Changes the last name validation to allow all characters so a user can submit their last name if they have `'` or `-`. 

#### How should this be reviewed?
👀 

#### Checklist
- [ ] Documentation added for changed endpoints.
- [ ] Tests added for new features/bug fixes.
- [ ] Post a message in #api if this includes something that causes a rebuild!  

---
For review: …

### CODE REVIEW

**Comment by DFurnes** [comment]
Before we merge this, can we [either update the existing test](https://github.com/DoSomething/northstar/blob/a41af99f9917f3a131f312354cbee696747b584c/tests/Http/Web/WebUserTest.php#L63) or add a new one to prove this is fixed by the change & catch any accidental regressions in the future?

**Comment by chloealee** [comment]
I think @DoSomething/team-bleed was added automatically as a Reviewer - is it ok to remove @DoSomething/team-bleed as an automatic reviewer?

**Comment by DFurnes** [comment]
Yes! Let's swap that out to @DoSomething/team-bleed 👍

**Comment by chloealee** [comment]
@DFurnes great idea! Just updated the test :)

**Comment by katiecrane** [comment]
YAY Northstar!! This is so exciting!!!

File: app/Http/Controllers/Web/UserController.php [hunk 1]
```diff
@@ -87,7 +87,7 @@ public function update(Request $request, $id)
 
         $this->registrar->validate($request, $user, [
             'first_name' => 'required|max:50',
-            'last_name' => 'nullable|alpha',
+            'last_name' => 'nullable',
```
Comment (DFurnes) [inline app/Http/Controllers/Web/UserController.php]: OH WAIT. Can we add a `max:50` on this field too, to prevent abuse?

Comment (chloealee) [reply app/Http/Controllers/Web/UserController.php]: added!

**Pull request merged**